mosaic/polypcolor.py

```diff
@@ -0,0 +1,75 @@
+from matplotlib.axes import Axes
+from matplotlib.collections import PolyCollection
+from matplotlib.colors import Normalize, Colormap
+from mosaic.descriptor import Descriptor
+
+from numpy.typing import ArrayLike
+
+from xarray.core.dataarray import DataArray
+
+
+def polypcolor(
+    ax: Axes,
+    descriptor: Descriptor,
+    c: ArrayLike,
```

Comment:
Doesn't `c` have to be an `xarray.DataArray`?  You're using `c.dims` with named dimensions, which suggests to me that it wouldn't work with, say, a `numpy.ndarray`.

Reply:
For right now `c` has be a `xarray.DataArray`. Eventually I'd like to have a little more flexibility and be able to compare the shape of `c` against the underlying patches. That way we'd be able to support `xarray.DataArray`, numpy data arrays, and dask data arrays. 

But I haven't done that yet, so I'll be more explicit in the typing for now.

Reply:
Sounds like a great plan!  Thanks for making the change for now.